env-titanium.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
 		'clearTimeout': false,
 		'clearInterval': false,
 		'setTimeout': false,
-		'setInterval': false
+		'setInterval': false,
+		'exports': false,
+		'module', false
```

Comment:
Hmm, interesting, but my syntax senses say this won't fly.

Reply:
Lol, wow I'm dumb changing now